kernel_test.go

```diff
@@ -15,6 +16,249 @@ const (
 	success = "\u2713"
 )
 
+const (
+	connectionKey = "a0436f6c-1916-498b-8eb9-e81ab9368e84"
+	sessionID     = "ba65a05c-106a-4799-9a94-7f5631bbe216"
+	transport     = "tcp"
+	ip            = "127.0.0.1"
+	shellPort     = 57503
```

Comment:
Can we parse the shell and iopub ports from the fixture connection file in the test init?  That way, if ports change for any reason we only have to change them in one place.